engine/config_integrations/alertmanager.py

```diff
@@ -22,26 +22,27 @@
 
 # Web
 web_title = """\
-{%- set groupLabels = payload.groupLabels.copy() -%}
-{%- set alertname = groupLabels.pop('alertname') | default("") -%}
+{%- set groupLabels = payload.get("groupLabels", {}).copy() -%}
+{%- set alertname = groupLabels.pop('alertname', "")  -%}
 
 
-[{{ payload.status }}{% if payload.status == 'firing' %}:{{ payload.numFiring }}{% endif %}] {{ alertname }} {% if groupLabels | length > 0 %}({{ groupLabels|join(", ") }}){% endif %}
-"""  # noqa
+[{{ payload.status }}{% if payload.status == 'firing' and payload.numFiring %}:{{ payload.numFiring }}{% endif %}] {{ alertname }} {% if groupLabels | length > 0 %}({{ groupLabels|join(", ") }}){% endif %}"""  # noqa
 
 web_message = """\
-{%- set annotations = payload.commonAnnotations.copy() -%}
+{%- set annotations = payload.get("commonAnnotations", {}).copy() -%}
+{%- set groupLabels = payload.get("groupLabels", {}) -%}
+{%- set commonLabels = payload.get("commonLabels", {}) -%}
+{%- set severity = groupLabels.severity -%}
 
-{% set severity = payload.groupLabels.severity -%}
 {% if severity %}
 {%- set severity_emoji = {"critical": ":rotating_light:", "warning": ":warning:" }[severity] | default(":question:") -%}
 Severity: {{ severity }} {{ severity_emoji }}
 {% endif %}
 
-{%- set status = payload.status | default("Unknown") %}
+{%- set status = payload.get("status", "Unknown") %}
```

Comment:
just curious, what is the difference here? I think previous version is more straightforward and clear for non-python developers

Reply:
Just aligned it with other `.get`usages. ( It was used in previous templates also)